src/main/java/uk/nhs/digital/nhsconnect/nhais/controller/AcceptanceController.java

```diff
@@ -4,18 +4,37 @@
 import ca.uhn.fhir.parser.IParser;
 import org.hl7.fhir.r4.model.OperationOutcome;
 import org.hl7.fhir.r4.model.Patient;
+import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.http.HttpStatus;
 import org.springframework.web.bind.annotation.*;
+import uk.nhs.digital.nhsconnect.nhais.model.edifact.TranslatedInterchange;
+import uk.nhs.digital.nhsconnect.nhais.model.mesh.MeshMessage;
+import uk.nhs.digital.nhsconnect.nhais.service.EdifactToMeshMessageService;
+import uk.nhs.digital.nhsconnect.nhais.service.FhirToEdifactService;
+import uk.nhs.digital.nhsconnect.nhais.service.OutboundMeshService;
 
 @RestController
 public class AcceptanceController {
 
+    @Autowired
+    private OutboundMeshService outboundMeshService;
+
+    @Autowired
+    private FhirToEdifactService fhirToEdifactService;
+
+    @Autowired
+    private EdifactToMeshMessageService edifactToMeshMessageService;
```

Comment:
instead of autowire we can use all args constructor:
@RestController
@AllArgsConstructor
public class AcceptanceController {
    private OutboundMeshService outboundMeshService;
    private FhirToEdifactService fhirToEdifactService;
    private EdifactToMeshMessageService edifactToMeshMessageService;